fuzz/fuzz_targets/packet.rs

```diff
@@ -0,0 +1,21 @@
+#![cfg_attr(not(windows), no_main)]
+
+#[cfg(not(windows))]
+use libfuzzer_sys::fuzz_target;
+
+#[cfg(not(windows))]
+fuzz_target!(|data: &[u8]| {
+    use neqo_transport::{packet::PublicPacket, RandomConnectionIdGenerator};
+
+    lazy_static::lazy_static! {
```

Comment:
Rather than once lazy_static, use OnceCell: https://docs.rs/once_cell/latest/once_cell/#faq (don't use the crate, it's in the std library now)

Reply:
I used `OnceLock`.